svn/common.py

```diff
@@ -171,56 +175,57 @@ def properties(self, rel_path=None):
         property_list = []
 
         result = self.run_command(
-                    'proplist', 
-                    ['--xml', full_url_or_path], 
-                    combine=True)
+            'proplist',
+            ['--xml', full_url_or_path],
+            combine=True)
 
         # query the proper list of this path
         root = xml.etree.ElementTree.fromstring(result)
         target_elem = root.find('target')
-        property_names = [ p.attrib["name"] for p in target_elem.findall('property')]
+        property_names = [p.attrib["name"]
+                          for p in target_elem.findall('property')]
 
         # now query the content of each propery
         property_dict = {}
 
         for property_name in property_names:
             result = self.run_command(
-                        'propget', 
-                        ['--xml', '{0}'.format(property_name), full_url_or_path, ], 
-                        combine=True)
+                'propget',
+                ['--xml', '{0}'.format(property_name),
```

Comment:
Please change `'{0}'.format(property_name)` to `property_name` while you're at it.